ja2003.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.github.christianrauch.Jahresarbeit-2003</id>
+  
+  <name>Jahresarbeit 2003</name>
+  <summary>Find die größte Fläche von zusammenhängenden Quadraten</summary>
+
+  <url type="homepage">https://github.com/christianrauch/Jahresarbeit-2003</url>
+  
+  <project_license>GPL-2.0</project_license>
+  <metadata_license>OFL-1.1</metadata_license>
+  <metadata_license>Apache-2.0</metadata_license>
```

Comment:
1. only one `metadata_license`
2. CC0-1.0 is prefered. I don't think any of those two is accepted.

Reply:
I may have misinterpreted what "metadata" is. Is this the AppData XML itself? Or is this for "resources" like images? How do I specify that a project is covered by multiple licences? E.g. GPL for code, OFL for fonts, etc.?

Reply:
You mentionned above in https://github.com/flathub/flathub/pull/2705#issuecomment-1001226835 "AppStream metadata". This is exactly that.